src/model/dataTypes/primitiveDataTypes.ts

```diff
@@ -67,6 +67,7 @@ export const mongoidPrimitiveTypes: { [name: string]: string } = {
   positiveInt: "Integer",
   question: "String",
   string: "String",
+  String: "String",
```

Comment:
What's going on here? We shouldn't have any primitive types with capital names, right?

Reply:
that's system type string. `System.String`. Either I've to add it here or add a condition in template to see if it is system type. If system type, don't find mapping from `mongoidPrimitiveTypes`

Reply:
OKay. that's wrong. I've to get rid of it. will fix it. Thank you!

Reply:
Done